### PR TITLE
Create a Pull Request Template

### PR DESCRIPTION
**Pull request by rmdias**
**This PR is solving a issue? Please provide the issue number (ex: issue #29)**
issue #3 ??

**What is this PR state?**
- [ ] pr_state: WIP
- [x] pr_state: PLEASE_REVIEW

**What kind of change does this PR introduce?** (check one with "x")
- [ ] Translation
- [x] Other... Please describe:

**Others informations**:
Pushing the PR Template file and Commit Message Guide

### CODE REVIEW

**Comment by rmdias** [comment]
I signed it!

File: .github/PR_Template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Thank you for submitting this PR!
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines: `trans(title): topic1, topic2 or review(title): topic1, topic2`
```
Comment (cironunes) [inline .github/PR_Template.md]: maybe we can put `context` instead of `title` inside the parenthesis

Comment (cironunes) [inline .github/PR_Template.md]: we should also have a commit message guideline markdown file so we can link it. #5

File: .github/PR_Template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] The translation content follows our commit conventions
+
+**This PR is solving a issue? Please provide the issue number (ex: issue #29)**
```
Comment (cironunes) [inline .github/PR_Template.md:7]: `Is the PR solving an issue? Please provide the number`

We should also make sure to include in the footer of the commit messages the `Closes #[number]`

File: .github/PR_Template.md (hunk 1 above)
Comment (rmdias) [inline .github/PR_Template.md]: Yes, I'll work on this today. @cironunes

**Comment by rmdias** [comment]
Check it please @cironunes

**Comment by willmendesneto** [comment]
@rmdias @cironunes Any updates? Are we using the [Angular commit messages convention](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit#heading=h.em2hiij8p46d)? If yes, we need to validate the commits message too?

**Comment by cironunes** [comment]
Please squash the commits into one so we can merge.

**Comment by cironunes** [comment]
bump @rmdias

**Pull request closed without merging**